40-LicenseRequestModel.inc.md

```diff
@@ -14,17 +14,15 @@ This license request model defines a mechanism for achieving both goals. This re
 * DASH clients can execute DRM workflows without [=solution-specific logic and configuration=].
 * Custom code specific to a license server implementation is limited to backend business logic.
 
-These benefits increase in value with the size of the solution, as they reduce the development cost required to offer playback of encrypted content on a wide range of DRM-capable client platforms using different [=DRM systems=], with [=licenses=] potentially served by different license server implementations.
```

Comment:
That's an important value statement explaining why the interoperable model has relevance. I think it adds value to the text. Perhaps we could further emphasize the affordances it gives to backend systems simplification but as a first step, this statement seemed worth having.

Reply:
Keep it.